src/ui/locales/zh-Hans.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "切换属性",
+    "AttributionControl.MapFeedback": "地图反馈",
+    "FullscreenControl.Enter": "进入全屏",
+    "FullscreenControl.Exit": "退出全屏",
+    "GeolocateControl.FindMyLocation": "查找我的位置",
+    "GeolocateControl.LocationNotAvailable": "位置不可用",
+    "LogoControl.Title": "Mapbox 徽标",
+    "Map.Title": "地图",
+    "NavigationControl.ResetBearing": "将方位重置为北",
```

Comment:
What does Bearing mean?

The Chinese translation means "Reset the location to North"

Reply:
@Jing-flyloveyin Bearing is the direction you're facing. 

For you reference, this is the original English strings to compare with https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/default_locale.js